src/cow-react/common/utils/fetch.ts

```diff
@@ -0,0 +1,28 @@
+import { backOff, BackoffOptions } from 'exponential-backoff'
+
+// See config in https://www.npmjs.com/package/@insertish/exponential-backoff
+const DEFAULT_BACKOFF_OPTIONS: BackoffOptions = {
+  numOfAttempts: 10,
+  maxDelay: Infinity,
+  jitter: 'none',
+}
+
+/**
+ *
+ * @param input Request info (as in the fetch method)
+ * @param init Request options (as in fetch method)
+ * @param backoffOptions Backoff parameters. By default It will wait a maximum of ~102s maximun wait time. Wait time is 100ms * 2**(attenmpt)
+ * @returns A promise of the request
+ */
+// By default, it will wait
+export function fetchWithBackoff(
+  input: RequestInfo | URL,
+  init?: RequestInit,
+  backoffOptions?: BackoffOptions
+): Promise<Response> {
+  if (backoffOptions) {
+    return backOff(() => fetch(input, init), { ...DEFAULT_BACKOFF_OPTIONS, ...backoffOptions })
+  }
+
+  return fetch(input, init)
```

Comment:
If I'm reading the code right, the backoff will only be used if we pass a third parameter to the fn `fetchWithBackoff`

As far as I could see in the other files, this is not the case, so in the end we'll still be using regular `fetch`.
![image](https://user-images.githubusercontent.com/43217/221167689-f717f3f2-d3df-41b1-9f5d-0a4988496d09.png)


Am I wrong?

Reply:
🙈 NVM, initially i was going to do the backoff to not happen by default. 

But I saw later there was no point on doing that. I forgot to get read of that. Fixed!